src/quo/components/wallet/token_input/style.cljs

```diff
@@ -24,7 +24,7 @@
           :margin-right (if platform/ios? 6 4)
           :color        (colors/theme-colors colors/neutral-100 colors/white theme)
           :padding      0
-          :text-align   :center
+          :flex         1
```

Comment:
@ulisesmac it seems you were also fixing the flickering as part of that PR https://github.com/status-im/status-mobile/pull/18520 which I think originally fixes anther issue -- to be able to add input in the middle. 

I see you did lots of changes to the style. I think that's the only needed change to fix the flickering issue

Reply:
Well, the flickering is solved by doing this, but it also affects the design.

Previously:
![image](https://github.com/status-im/status-mobile/assets/90291778/df690b42-1faf-4186-9aee-935abf92ef69)

In this PR:
![image](https://github.com/status-im/status-mobile/assets/90291778/443f1336-f1f8-4390-8c51-5be630af9858)

The "ETH" is always at the end.

Although it seems like a small thing to solve, it has been actually hard to keep the design while solving the issue.

Regarding to 
- https://github.com/status-im/status-mobile/issues/18498

Maybe we should close it since it's a duplicate of 
- #18458

BTW, I had an error setting the "fixes #..." in the PR you mentioned.
I attempted to fix `#18458` instead of `#18459` (one ends with `8`, the other one ends with `9`)

Reply:
Thanks for the clarification @ulisesmac. Totally, missed the "ETH" placement.

I am closing this PR then